pages/index.html

```diff
@@ -333,6 +333,40 @@ <h2 class="section-header">Academic at heart, 100% free software</h2>
   </div> <!-- section -->
 </div> <!-- container -->
 
+<!-- Precice support program  -->
+
+<div class="background-dark">
+  <div class="container">
+    <div class="section">
+
+      <div class="row">
+        <div class="col-lg-12">
+          <h2 class="section-header">You support preCICE - preCICE supports you</h2>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-lg-12">
+          <p>As the developers of preCICE, we enjoy supporting our academic and industry users, but due to the growing demand, we are not able to service all support requests any more. We are introducing <strong>'preCICE support'</strong> as a way to contribute to sustainable open source software development and ensure preCICE is developed and maintained in the future. Supporting preCICE comes with several benefits:</p>
```

Comment:
```suggestion
          <p>As the developers of preCICE, we enjoy supporting our academic and industry users, but due to the growing demand, we are not able to service all support requests any more. We are introducing <strong>preCICE support</strong> as a way to contribute to sustainable open-source software development and to ensure preCICE is developed and maintained in the future. Supporting preCICE comes with several benefits:</p>
```
Single quotes look strange to me. If we emphasize with boldface, this is already enough to separate the text and make it clear it is a name.